packages/.eslintrc.json

```diff
@@ -22,6 +31,10 @@
         "radix": "error",
         "eqeqeq": ["error", "always"],
         "prefer-regex-literals": "error",
+        "no-useless-concat": "error",
+        "no-throw-literal": "error",
+        "no-script-url": "error",
+        "@dimensiondev/no-number-constructor": "error",
```

Comment:
Why disallow the following code? Is there an auto-fix for this rule? What do you suggest to use?

```js
Number(expr)
```

Reply:
CC @guanbinrui